entities.py

```diff
@@ -0,0 +1,44 @@
+"""
+LibrePCB S-expression entities
+"""
+
+
+class Position(object):
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+    def to_s_exp(self) -> str:
+        return '(position {} {})'.format(self.x, self.y)
```

Comment:
We should probably apply [`format_float`](https://github.com/LibrePCB/librepcb-parts-generator/blob/master/common.py#L44) to all floats that we write to an s-expression.

Reply:
Fixed.

Reply:
Uhm, nope :smile:

Reply:
Oh man the only case were I didn't fix it was the one were the comment was :laughing:

Reply:
Fixed. For real I hope :wink: